.circleci/config.yml

```diff
@@ -211,7 +211,7 @@ jobs:
       - attach_workspace: *attach_options
       - run:
           name: Restore dependencies
-          command: pnpm install --prefer-frozen-lockfile
+          command: pnpm install --prefer-frozen-lockfile && pnpm run build --filter raiden-ts
```

Comment:
Why chain the command here? This will just clutter the output of the run and makes it harder to identify issues.

Reply:
The build was to be run in the `install` phase, it's intended for that as well, but for some unknown reason, `pnpm` on `circleCI` called but did nothing to build it. This chained command is just a complement to that, and it makes no sense to have it in its own `run`, and also, no one checks these preparetory steps outputs anyway unless something goes wrong, and it isn't too much output, some ~30L more.